test/integration-test.py

```diff
@@ -310,15 +310,72 @@ def run_certificates_per_name_test():
             print("\nCertificates per name test: expected %s not present in output" % s)
             die(ExitStatus.Error)
 
-@atexit.register
-def cleanup():
-    import shutil
-    shutil.rmtree(tempdir)
-    if exit_status == ExitStatus.OK:
-        print("\n\nSUCCESS")
-    else:
-        if exit_status:
-            print("\n\nFAILURE %d" % exit_status)
+default_config_dir = os.environ.get('BOULDER_CONFIG_DIR', '')
+if default_config_dir == '':
+    default_config_dir = 'test/config'
+
+def run_admin_revoker_test():
+    email_addr = "js.integration.test@letsencrypt.org"
+    cert_file = os.path.join(tempdir, "ar-cert.der")
+    cert_file_pem = os.path.join(tempdir, "ar-cert.pem")
+    key_file = os.path.join(tempdir, "key.pem")
+    # Issue certificate for serial-revoke test
+    if subprocess.Popen('''
+        node test.js --email %s --domains ar-test.com \
+          --certKey %s --cert %s --challType http-01 && \
```

Comment:
I think we can leave out the email, certKey, can challType arguments here and below.